sdks/python/apache_beam/dataframe/frames.py

```diff
@@ -27,12 +27,14 @@
 from apache_beam.dataframe import partitionings
 
 
-@frame_base.DeferredFrame._register_for(pd.Series)
-class DeferredSeries(frame_base.DeferredFrame):
+class DeferredDataFrameOrSeries(frame_base.DeferredFrame):
```

Comment:
Should we call it `DeferredNDFrame` to be consistent with pandas?

Reply:
I don't want to commit ourselves to being quite that generic at the moment.